packages/jest-types/__typetests__/jest.test.ts

```diff
@@ -248,9 +247,39 @@ expectType<void>(jest.setSystemTime(new Date(1995, 11, 17)));
 expectError(jest.setSystemTime('1995-12-17T03:24:00'));
 
 expectType<typeof jest>(jest.useFakeTimers());
-expectType<typeof jest>(jest.useFakeTimers('modern'));
-expectType<typeof jest>(jest.useFakeTimers('legacy'));
-expectError(jest.useFakeTimers('latest'));
+expectType<typeof jest>(jest.useFakeTimers({advanceTimers: true}));
+expectType<typeof jest>(jest.useFakeTimers({advanceTimers: 10}));
+expectType<typeof jest>(jest.useFakeTimers({doNotFake: ['Date']}));
+expectType<typeof jest>(
+  jest.useFakeTimers({
+    doNotFake: [
+      'Date',
+      'hrtime',
+      'nextTick',
+      'performance',
+      'queueMicrotask',
+      'requestAnimationFrame',
+      'cancelAnimationFrame',
+      'requestIdleCallback',
+      'cancelIdleCallback',
+      'setImmediate',
+      'clearImmediate',
+      'setInterval',
+      'clearInterval',
+      'setTimeout',
+      'clearTimeout',
+    ],
+  }),
+);
+expectType<typeof jest>(jest.useFakeTimers({legacyFakeTimers: true}));
+expectType<typeof jest>(jest.useFakeTimers({now: 1483228800000}));
+expectType<typeof jest>(jest.useFakeTimers({now: Date.now()}));
+expectType<typeof jest>(jest.useFakeTimers({now: new Date(1995, 11, 17)}));
+expectType<typeof jest>(jest.useFakeTimers({timerLimit: 1000}));
+expectError(jest.useFakeTimers({enableGlobally: true}));
+expectError(jest.useFakeTimers({enableGlobally: true, legacyFakeTimers: true}));
+expectError(jest.useFakeTimers({legacyFakeTimers: true, toFake: ['Date']}));
```

Comment:
error for `toFake` without unknown timer? e.g. `jest.useFakeTimers({toFake: ['foobar']})`